src/pages/services-jeunes/index.page.test.tsx

```diff
@@ -131,83 +140,220 @@ describe('Page Services Jeunes', () => {
 		});
 
 		describe('Si des services jeunes sont récupérés', () => {
-			it('affiche au maximum 6 services initialement', () => {
-				// Given
-				const serviceJeuneList = [
-					aServiceJeune({ titre: 'service 1' }),
-					aServiceJeune({ titre: 'service 2' }),
-					aServiceJeune({ titre: 'service 3' }),
-					aServiceJeune({ titre: 'service 4' }),
-					aServiceJeune({ titre: 'service 5' }),
-					aServiceJeune({ titre: 'service 6' }),
-					aServiceJeune({ titre: 'service 7' }),
-				];
-				const analyticsService = aManualAnalyticsService();
-
-				// When
-				render(
-					<DependenciesProvider analyticsService={analyticsService}>
-						<ServicesJeunePage serviceJeuneList={serviceJeuneList} />
-					</DependenciesProvider>,
-				);
-
-				// Then
-				const mesuresJeunesSection = screen.getByRole('region', { name: 'les services jeunes' });
-				const servicesJeunesList = within(mesuresJeunesSection).getAllByRole('listitem');
-				expect(servicesJeunesList.length).toBe(6);
+			describe('Sélection des filtres', () => {
+				describe('aucun filtre n’est renseigné', () => {
+					it('affiche l’ensemble des types de services', () => {
+						// Given
+						const serviceJeuneList = [
+							aServiceJeune({ categorie: ServiceJeune.Categorie.ACCOMPAGNEMENT, titre: 'service 1' }),
+							aServiceJeune({ categorie: ServiceJeune.Categorie.AIDES_FINANCIERES, titre: 'service 2' }),
+							aServiceJeune({ categorie: ServiceJeune.Categorie.ENGAGEMENT, titre: 'service 3' }),
+							aServiceJeune({ categorie: ServiceJeune.Categorie.ENTREE_VIE_PROFESSIONELLE, titre: 'service 4' }),
+							aServiceJeune({ categorie: ServiceJeune.Categorie.LOGEMENT, titre: 'service 5' }),
+							aServiceJeune({ categorie: ServiceJeune.Categorie.ORIENTATION_FORMATION, titre: 'service 6' }),
+						];
+						const analyticsService = aManualAnalyticsService();
+
+						// When
+						render(
+							<DependenciesProvider analyticsService={analyticsService}>
+								<ServicesJeunePage serviceJeuneList={serviceJeuneList} />
+							</DependenciesProvider>,
+						);
+
+						// Then
+						const mesuresJeunesSection = screen.getByRole('region', { name: 'les services jeunes' });
+						const servicesJeunesList = within(mesuresJeunesSection).getAllByRole('list')[1];
```

Comment:
C'est un peu arbitraire le `getAllByRole()[1]`
```suggestion
						const [/* tags */, servicesJeunesList] = within(mesuresJeunesSection).getAllByRole('list');
```
Le commentaire autour de `tags`, c'est pour le linter, je crois qu'il n'est pas content avec la variable inutilisée sinon

Reply:
Et avec le select dans le form, tu peux l'extraire en dehors de la section et du coup t'as plus qu'une liste